notebooks/Stormer-Verlet.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "932f3d29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up required packages\n",
+    "using Pkg\n",
+    "Pkg.activate(@__DIR__)\n",
+    "Pkg.instantiate()"
```

Comment:
We should also include the `Project.toml` and `Manifest.toml` files to ensure reproducibility with exactly the same package versions.